BritneyS/SparkleSpin
# Launch screen

## What you did :question:
In this PR, I added a new assets folder called 'Images' to hold the image assets needed for the launch screen, a gradient background and light logo, as well as a dark logo to use on the Home Screen (according to the current prototype). 

This PR also includes the styling of the launch screen (added through LaunchScreen.storyboard).

## How to test it :microscope:
- Run the app
- Note that the launch screen displays as expected (see screenshots!)

Testing notes:
- To pause on the launch screen, add a break point at the `didFinishLaunchingWithOptions` delegate method in `AppDelegate`.
- Feel free to test different device screens using the simulator! iPhone XR, iPhone X, and iPad Pro simulated screens are shown in the screenshots below.
![screen shot 2019-02-21 at 4 31 12 pm](https://user-images.githubusercontent.com/8409475/53203293-c384d480-35f6-11e9-883d-178703a79f81.png)


## Any background context you want to provide? :question:
I'm still seeking feedback on the prototype, so this PR may be updated if changes are proposed. Please review PR https://github.com/BritneyS/SparkleSpin/pull/1 before this one!


## Screenshots (if applicable) :camera:
**Updated 2-27-19: Updated screenshots to reflect gradient direction change from horizontal to diagonal.**
![simulator screen shot - iphone xr - 2019-02-27 at 10 29 41](https://user-images.githubusercontent.com/8409475/53501983-27861d80-3a7b-11e9-8401-d64e9ccea0c2.png)
![simulator screen shot - iphone x - 2019-02-27 at 10 30 22](https://user-images.githubusercontent.com/8409475/53502000-2fde5880-3a7b-11e9-8b9a-426eb8f782c0.png)
![simulator screen shot - ipad pro 12 9-inch 3rd generation - 2019-02-27 at 10 31 04](https://user-images.githubusercontent.com/8409475/53502014-37056680-3a7b-11e9-9381-43cf1a4510f2.png)
